### DISCUSSION TITLE
[Announcement] Threat Hunt

### DISCUSSION THREAD

**Discussion by sujin0529** [prosyslab-classroom/cs348-information-security#385] [Announcements]
Students must sign a security agreement form to participate in the **"Threat Hunt"** activity.
We will distribute the agreement form next **Monday (03/13)** and **Wednesday (03/15)** in class.
Please read the agreement form carefully and sign it.
If you do not sign the agreement form, your submission will not be graded.

## Submission
- Write the details of the inspection time and the IP that you use to inspect in the result report.
- Attach threat hunt demo video and result report to email.
- Send cs348.hack@kaist.ac.kr an email whose title starts with **[2023-정보보호개론]** or **[2023-InformationSecurity]**.

![image](https://user-images.githubusercontent.com/44044134/224248940-c8c2ed96-99be-45bb-829a-8f642b58d99b.png)

Thanks.

**Comment by yeonhee-ryou** [comment]
Please add the device and OS information in the report or email.

## Email format
* Title: [2023-InformationSecurity] (title of your work)
* Contents
  * IP
  * Inspection time
  * Device: (ex: PC, Smart Phone, Tab, etc.)
  * OS: (ex: Windows 10, Centos 7, Android 11, IOS 16, etc.)
* Attachments: demo video and result report

**Comment by yeonhee-ryou** [comment]
## Additional Announcement for Threat Hunt

Hi, all
We are thankfully receiving a lot of reports about threat hunt.
But, we would like to announce some restrictions to keep this activity going for a long time.

- Do not send the report with the same root cause.
  - We'll grade it with your first report if you send multiple reports of the same root cause.
- Do not upload data used in threat hunt activity outside the KAIST domain.
- Do not practice DDoS Attack.
- Do not falsify the DB data.

Thanks for participating in the threat hunt.
